plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanIssueDetailsPanel.kt

```diff
@@ -71,6 +72,9 @@ internal class CodeWhispererCodeScanIssueDetailsPanel(
         )
         editorPane.revalidate()
         editorPane.repaint()
+        SwingUtilities.invokeLater {
```

Comment:
`runInEdt` or `ApplicationManager.getApplication().invokeLater`. Please read the documentation block for the latter for an explaination

also note that the block inside the closure is run async if that is not that expectation